modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/Commons.java

```diff
@@ -392,7 +392,7 @@ public static <T> T compile(Class<T> interfaceType, String body) {
 
             return (T) cbe.createInstance(new StringReader(body));
         } catch (Exception e) {
-            throw new SqlException(EXPRESSION_COMPILATION_ERR, e);
+            throw new SqlException(INTERNAL_ERR, "Unable to compile expression", e);
```

Comment:
any code from error group `Common` should be used with Ignite[Internal][Checked]Exception

Reply:
Fixed.